src/Analysis/Ast/Impl/Analyzer/PythonAnalyzer.cs

```diff
@@ -123,6 +123,8 @@ public sealed class PythonAnalyzer : IPythonAnalyzer, IDisposable {
             lock (_syncObj) {
                 _analysisEntries.Remove(new AnalysisModuleKey(module));
             }
+
+            _dependencyResolver.Remove(new AnalysisModuleKey(module));
```

Comment:
Looks we new it every time. Minor thing, but maybe just attach it to the module or have it produce it? Up to you.